configure

```diff
@@ -2750,7 +2750,7 @@ cat >>$CONFIG_STATUS <<_ACEOF || ac_write_fail=1
 _ACAWK
 cat >>"\$ac_tmp/subs1.awk" <<_ACAWK &&
   for (key in S) S_is_set[key] = 1
-  FS = ""
+  FS = "$(printf '\a')"
```

Comment:
whats this about?

Reply:
I couldn't build it on my Mac machine, so I'm building it on my Windows machine with WSL. The Windows files have CRLF in them but the Linux interpreters expect normal unix-style newlines, so I need to use `dos2unix` on some of the files, including `configure`, to get it to run properly. But `dos2unix` refuses to do anything if the file contains a raw bell character in the source code, so I want to change it to something like `"$(printf '\a')"` which should evaluate to the same thing while being friendly to `dos2unix`.

Reply:
heh, ok.